tests/playbooks/roles/install-csi-cinder/tasks/main.yaml

```diff
@@ -142,7 +142,7 @@
       set -e
       set -o pipefail
 
-      cd $GOPATH/src/k8s.io/cloud-provider-openstack
+      cd $GOPATH/src/k8s.io/cloud-provider-openstack/tests/e2e
       mkdir /var/log/csi-pod
       go test -v ./cmd/tests/cinder-csi-e2e-suite/cinder_csi_e2e_suite_test.go -ginkgo.v -ginkgo.progress -ginkgo.skip="\[Disruptive\]" -ginkgo.focus="\[cinder-csi-e2e\]" -ginkgo.noColor -timeout=0 -report-dir="/var/log/csi-pod" | tee "/var/log/csi-pod/cinder-csi-e2e.log"
```

Comment:
will the folder need update ?

Reply:
Yes, because the go tools expect commands like build and  run to be executed within a module.

I'm also thinking about moving to use upstream kubernetes e2e.test to replace our binary when I find some time. It is configurable, so that the same set of tests will run.